llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -1393,6 +1396,108 @@ class GetElementPtrInst final : public Instruction {
 #endif
 };
 
+class AllocaInst final : public UnaryInstruction {
+  Use getOperandUseInternal(unsigned OpIdx, bool Verify) const final {
+    return getOperandUseDefault(OpIdx, Verify);
+  }
+  SmallVector<llvm::Instruction *, 1> getLLVMInstrs() const final {
```

Comment:
perhaps we should have a helper `SingleLLVMInstructionImpl` helper class for these that you can inherit from

Reply:
Yeah that's a good point, let me give it a try.

Reply:
I think that would also help with a couple more functions that repeat in a similar way, like `getOpreandUseInteral()`, `getUseOperandNo()` and `getNumOfIRInstrs()`.

I will upload a separate patch for this.